common/src/main/java/com/wynntils/models/territories/AuraTimerModel.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.territories;
+
+import com.wynntils.core.components.Model;
+import com.wynntils.mc.event.SubtitleSetTextEvent;
+import java.util.List;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+// NOTE: As the package suggests, this is model is related to guild wars' Aura.
+//       Not to be confused with the aura spell.
+public class AuraTimerModel extends Model {
```

Comment:
I don't really know what this is, but the title string indicates it could be called "tower aura". Maybe we can rename the class TowerAuraTimerModel?

Reply:
It is guild tower aura yes, so I can rename this.